build/prerequisites.ps1

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
maybe this file should be named `install-prereqs.ps1` to keep it consistent.

Reply:
Because it at the root I thought it might do more varied stuff than the individual components that only need to install prereqs. Actually, I'll rename them all to prerequisites.ps1...